Dockerfile

```diff
@@ -35,14 +41,20 @@ COPY pkg/ pkg/
 COPY util/ util/
 
 # Build
+ARG package=.
 ARG ARCH
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GO111MODULE=on go build -ldflags "-extldflags '-static'" -a -o manager main.go
+ARG LDFLAGS
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.local/share/golang \
+    CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GO111MODULE=on go build -ldflags "${LDFLAGS} -extldflags '-static'"  -o manager ${package}
+ENTRYPOINT [ "/start.sh", "/workspace/manager" ]
```

Comment:
This is just an intermediate container, is setting `ENTRYPOINT` here will make any difference?

Reply:
Removed not required.